lib/internal/inspector/inspect_repl.js

```diff
@@ -375,7 +376,10 @@ function createRepl(inspector) {
 
     [util.inspect.custom](depth, { stylize }) {
       const { startTime, endTime } = this.data;
-      return stylize(`[Profile ${endTime - startTime}μs]`, 'special');
+      return stylize(
+        `[Profile ${endTime - startTime}${String.fromCharCode(956)}s]`,
```

Comment:
It would probably be useful to have a constant `MU = String.fromChar(956)` or something so someone reading the code doesn't have to know what that char `956` – and probably worth to add a comment explaining that we don't include non-ASCII char in the source code for a reason.

Reply:
Good idea. Done. Thanks.